pgxctx/example_test.go

```
@@ -0,0 +1,32 @@
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func Example() {
+	logger := log.NewLogfmtLogger(log.NewSyncWriter(os.Stdout))
+
+	cfg, err := pgxpool.ParseConfig("postgres://example.com")
+	if err != nil {
+		panic(err)
+	}
+
+	cfg.ConnConfig.Tracer, err = pgxctx.NewLogger(
```

golangci-lint (2.12.2)

[error] ineffectual assignment to err

(ineffassign)